typescript/ws-protocol/src/FoxgloveServer.test.ts

```diff
@@ -480,4 +487,83 @@ describe("FoxgloveServer", () => {
     }
     close();
   });
+
+  it.each([
```

Comment:
nit: I think you can do something like `it.each<FetchAssetResponse>(...)` and remove `as`